src/components/Homepage/Carousel/Item.astro

```diff
@@ -3,31 +3,38 @@ import { Image } from "astro:assets";
 import tickIcon from "../../../assets/icons/tick.svg";
 import "./ItemFrame.css";
 
-interface Props {
-  type?: "text" | "image";
-  message?: string;
+interface Item {
+  type: string;
   source: string;
-  imageSrc?: ImageMetadata;
-  imageAlt?: string;
-  bottomWidth?: number;
-  padding?: number;
+  bottomWidth?: number; // Value in rem
+  padding?: number; // Value in rem
```

Comment:
Sorry, I forgot to say in my original message, but it's considered a best practice to include units in identifier names. For example, in this case, `bottomWidthRem` and `paddingRem`.

To be clear, I'm only talking about the names. The values are fine as `number`s.

These comments won't be seen where these values are specified or used.